Gemfile.lock

```diff
@@ -17,7 +17,7 @@ GEM
     kgio (2.10.0)
     method_source (0.8.2)
     mime-types (1.25.1)
-    mixlib-install (3.11.21)
```

Comment:
Let's make sure to get a pin the gemfile to >= the necessary version so we don't have some weird depsolving mess that reverts this later on. It's happened more than once in various projects.

Reply:
Done!